crates/networking/p2p/sync.rs

```diff
@@ -615,16 +616,20 @@ impl Syncer {
                     .any(|(ch, end)| ch < end)
             })
         {
+            let storage_trie_rebuilder_sender;
+            if let Some(trie_rebuilder) = self.trie_rebuilder.as_ref() {
+                storage_trie_rebuilder_sender = trie_rebuilder.storage_rebuilder_sender.clone();
+            } else {
+                info!("Storage trie rebuilder sender is None");
```

Comment:
This is a very implementation specific log. Logs (specifically INFO logs) are meant more for users than devs.